src/Display/Slider/Slider.tsx

```diff
@@ -201,11 +201,17 @@ Slider.Item = SliderItem;
 export interface Props {
   children: React.ReactNode;
   className?: string;
+  /** The index of the page that will be shown initally (before the first manual- or auto-scroll). Note that the index starts at 1. */
```

Comment:
The initialItem can have the default value as 1, then we don't need the note, right?

Reply:
I think still important to note, because every other index that I've ever seen is 0-based, not 1-based.